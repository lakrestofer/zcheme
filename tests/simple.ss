(loop for x in '(1 2 3)
      do (print x))
