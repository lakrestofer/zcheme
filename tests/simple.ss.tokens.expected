Token { .kind = Lexer.TokenKind.l_paren, .start = 0, .end = 1 }
Token { .kind = Lexer.TokenKind.identifier, .start = 1, .end = 5 }
Token { .kind = Lexer.TokenKind.identifier, .start = 6, .end = 9 }
Token { .kind = Lexer.TokenKind.identifier, .start = 10, .end = 11 }
Token { .kind = Lexer.TokenKind.identifier, .start = 12, .end = 14 }
Token { .kind = Lexer.TokenKind.quote, .start = 15, .end = 16 }
Token { .kind = Lexer.TokenKind.l_paren, .start = 16, .end = 17 }
Token { .kind = Lexer.TokenKind.integer, .start = 17, .end = 18 }
Token { .kind = Lexer.TokenKind.integer, .start = 19, .end = 20 }
Token { .kind = Lexer.TokenKind.integer, .start = 21, .end = 22 }
Token { .kind = Lexer.TokenKind.r_paren, .start = 22, .end = 23 }
Token { .kind = Lexer.TokenKind.identifier, .start = 30, .end = 32 }
Token { .kind = Lexer.TokenKind.l_paren, .start = 33, .end = 34 }
Token { .kind = Lexer.TokenKind.identifier, .start = 34, .end = 39 }
Token { .kind = Lexer.TokenKind.identifier, .start = 40, .end = 41 }
Token { .kind = Lexer.TokenKind.r_paren, .start = 41, .end = 42 }
Token { .kind = Lexer.TokenKind.r_paren, .start = 42, .end = 43 }
